### ISSUE TITLE
Markdown emoji code may not display well inside fenced code blocks

### ISSUE THREAD

**Issue by Yarakashi-Kikohshi** [simov/markdown-viewer#214]
Thank you very much for the very nice extension. 👍 

If you write Markdown emoji code inside fenced code blocks with a specified language, the code will not display well.

## To reproduce

Consider three cases. `:smile:` doesn't display well in fenced code blocks with a specified language.

- If `markdown` is specified.
    ````markdown
    ```markdown
    smile emoji → :smile:
    ```
    ````
    ![image](https://github.com/simov/markdown-viewer/assets/75878492/c892b5a3-879e-4f0a-9d82-72366d850f1b)
- If languages other than `markdown` is specified, e.g. `python`.
    ````markdown
    ```python
    print(":smile:")
    ```
    ````
    ![image](https://github.com/simov/markdown-viewer/assets/75878492/145c8c91-450e-4e8f-9694-66dc884ae2f9)
- If no language is specified.
    ````markdown
    ```
    :smile:
    ```
    ````
    ![image](https://github.com/simov/markdown-viewer/assets/75878492/ed194afd-e3b5-4b63-a270-963f81150ba3)

This issue does not occur when trying to display non-existent Markdown emoji code.

````
```markdown
:no_emoji:
```
````

![image](https://github.com/simov/markdown-viewer/assets/75878492/43410d27-c500-4ba8-8a29-743e8e06ffd4)

## Version

- Markdown Viewer: 5.1
- Microsoft Edge: 113.0.1774.50

**Comment by simov** [comment]
This bug got fixed in v5.3, let me know if you find anything else.

**Comment by yarakos95** [comment]
I am sorry for the late reply.

Thank you for resolving the issue! :tada:

Other than this issue, the v5.3 update is very attractive. I love this extension even more.
